entity-view/impl/src/main/java/com/blazebit/persistence/view/impl/mapper/ViewMapper.java

```diff
@@ -185,26 +187,38 @@ private ObjectMapper createAccessor(ManagedViewType<S> sourceType, ManagedViewTy
         }
 
         // Try to find a source attribute
-        MethodAttribute<? super S, ?> sourceAttribute = sourceType.getAttribute(targetAttribute.getName());
-
-        if (sourceAttribute == null) {
-            if (targetAttribute.getMappingType() == Attribute.MappingType.PARAMETER) {
-                return new ParameterObjectMapper(((MappingAttribute<?, ?>) targetAttribute).getMapping());
-            }
-            // Optionally ignore missing attributes
-            if (ignoreMissing) {
+        MethodAttribute<?, ?> sourceAttribute;
+        AttributeAccessor accessor;
+        if (sourceType == null) {
+            sourceAttribute = null;
+            accessor = Accessors.forEntityMapping((EntityViewManagerImpl) entityViewManager, targetAttribute);
+            if (ignoreMissing || (targetAttribute.getMappingType() == Attribute.MappingType.PARAMETER)) {
                 return null;
+            } else if (accessor == null) {
+                throw inconvertible("Attribute '" + targetAttribute.getName() + "' from target type is missing in source type!", targetType);
+            }
```

Comment:
IMO parameter mappings should be supported and null should only be returned if the accessor is null.

```suggestion
            if (targetAttribute.getMappingType() == Attribute.MappingType.PARAMETER) {
                return new ParameterObjectMapper(((MappingAttribute<?, ?>) targetAttribute).getMapping());
            }
            accessor = Accessors.forEntityMapping((EntityViewManagerImpl) entityViewManager, targetAttribute);
            if (accessor == null) {
                if (ignoreMissing) {
                    return null;
                }
                throw inconvertible("Attribute '" + targetAttribute.getName() + "' from target type is missing in source type!", targetType);
            }
```